card-flow.md

```diff
@@ -0,0 +1,57 @@
+
+## Board Flow
+
+In Kanban, we add cards on the far left-most column that are to be worked on by the team. As cards are ready to be worked on and there's availability by the team, we move the cards to the right as they go through the steps of software development.  Each column represents the stages that a specific project has defined.  We will illustrate our common flow below but each project may modify this based on business conditions.
+
+### Where's the backlog or icebox columns?
+
+Some tools have a place to keep cards that don't 'fit' into the current iteration or cycle of work.  This concept doesn't exist in Kanban and it's been our experience that trying to mimic either backlogs or iceboxes lead to out-dated and wasteful cards that ultimately slow down the team.  Instead we have the client keep any extra work not quite ready to be tackled elsewhere.  This allows only 'fresh' ideas to enter the board to be worked on by the team in the next 1-2 weeks at most.
+
+### The "Next" Column
+Once cards are approved and ready for developers to begin working on them, we move them to the "Next" column. Cards in this column are our queue of work, and are waiting for someone to be assigned to them. A developer can assign themselves to a card when they are ready to begin work on it.
+
+### The "Development" Column
+
+The "Development" column is for cards that are actively being worked on. It is important that a developer has only one card in this column at a time. If they need to work on something else, this card can move back to ready so as not to let things build up or block. Our project board system follows the traditional Kanban pull based system pretty closely. Things can only move right if there's space for them, and a dev team can really only take on 3-4 cards at once in the development stage. More than that and things sit idle. The board should be an accurate reflection of the project at any point in time, so if a developer is not currently working on a card in the "Development" column, they can move it back so our progress is accurately reflected.
+
+### The "Needs Review" Column
+
+When a developer has completed the task on a card, we kick off the (pull request process)[https://github.com/HaughtCodeworks/TeamGuide]. The "Needs Review" column represents cards currently under code review in the PR process. After a PR has been fully approved, the feature is merged into the master branch and deployed to the staging environment.
+
+### The "Staging" Column
+
+Cards in the "Staging" column are ready for client review on the staging environment. The client makes sure everything on the feature works as expected.
+
+### The "Approved" Column
+
+Once clients have tried out a feature and approved a card in "Staging", they move the card to the "Approved" column. Cards that are approved will be promoted to production at the next opportunity.
```

Comment:
Let's add a bit more clarity to this line:

Cards that are approved will be promoted to production at the next opportunity.

instead let's go with:

Cards in this column are ready to be promoted to production. Each project will have its own rules on when we deploy to production so make sure these get scheduled and communicated with the client and team before they're deployed.